docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -145,9 +145,27 @@ the work and use the terms we find most accurate and inclusive.
 
 1.2 Glossary of General Terminology
 -----------------------------------
-In this section, we will create a glossary of general terminology that will be used 
-throughout this style guide (e.g., for starters, we should explicitly define sex and 
-gender; SGM) (include visual of 'Do's and Don'ts)
+In this section, you will find a glossary of general terminology that will be used 
+throughout this style guide. 
+
+
+.. list-table:: **Glossary**
+   :header-rows: 1
+
+   * - Term
+     - Definition
+   * - Sex
+     - *Sex* refers to biological attributes and functions associated with chromosomal makeup, endocrine function, reproductive organs, secondary sex characteristics, and other physiological and morphological variations and processes not directly related to reproduction (e.g., body size and composition, metabolic function, or organ function) [Ritz_and_Greaves_2022]_.
+   * - Gender
+     - *Gender*, on the other hand, is a sociocultural construct that refers to social norms, behaviors, and institutional structures that shape any individual's experiences, exposures, and access to power and resources, as well as one's internal sense of self as a gendered individual [Ritz_and_Greaves_2022]_. 
+   * - Sexual and gender minority (SGM) 
+     - An umbrella term that includes (but is not limited to) individuals that are two-spirit, intersex, lesbian, gay, bisexual, transgender, non-binary, asexual, or have other gender non-conforming identities.
+   * - Nondifferentiated sex/gender measure
+     - A useful hybrid term that reflects the dynamic relationship that exists between sex and gender and acknowledges the common conflation between sex and gender in public health research and data [Ritz_and_Greaves_2022]_. 
+
```

Comment:
I think this is good! But I think we could expand. First, we could add more BLISS terms: intersex, non-binary, transgender, sex assigned at birth, cisgender, gendered language etc. We also could include some more research/simulation terms, but this is probably not needed if the audience is IHME people only, simulation, primary data, proxy measure, etc.

Reply:
Oh yes good point! Sounds great. I like to think this could somehow have an audience beyond just IHME people, but maybe that's out of the scope for now